pyapp/{{ cookiecutter.slug }}/{{ cookiecutter.package_name }}/app.py

```diff
@@ -0,0 +1,14 @@
+import logging
+import time
+
+log = logging.getLogger(__name__)
+
+
+def main():
+    return "Hello, world!"
+
+
+if __name__ == "__main__":  # pragma: no cover
+    while True:
+        log.info(main())
+        time.sleep(1)
```

Comment:
Just some arbitrary continuously-running code so that there's something for `make dev` and `make docker-run` to run without stopping